tests/test_renku.py

```diff
@@ -44,8 +45,19 @@ def test_notebook_launch(browser):
     browser.fill('username', 'cramakri')
     browser.fill('password', 'cramakri')
     browser.find_by_id('kc-login').click()
+
+    # wait a bit for the page to load, helps to avoid test failures
+    time.sleep(2)
```

Comment:
I don't think this is needed

Reply:
any guesses how this `time.sleep()` affects the behaviour of the above code? just saying...

Reply:
@fgeorgatos see the comment from @ableuler below